anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/util/ModelServiceConnector.java

```diff
@@ -146,4 +150,29 @@ public ModelResource findLatestModel(UUID detectorUuid) {
 
         return modelResourceList.get(0);
     }
+
+    public DetectorResources findUpdatedDetectors(int timePeriod) {
+        isTrue(timePeriod > 0, "timePeriod can't be negative");
```

Comment:
Consider "timePeriod must be strictly positive". The current error message allows 0, but in fact that's not allowed.